src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -138,9 +138,11 @@ const ShareTrajectoryModal = ({
         errorMessage: {
             content: (
                 <div>
-                    <h4>{Warn} The current file is stored on your device.</h4>
-                    <div>
-                        <h5>
+                    <h5 className={styles.localFileHeading}>
+                        {Warn} The current file is stored on your device.
+                    </h5>
```

Comment:
Is `h5` just being used for styling here? Is there an `h4` element above this somewhere in the HTML hierarchy?